### PR TITLE
Create health_data.md

### PR DESCRIPTION
**Pull request by smza**
Transferring the content for Health data page from  a google doc

### CODE REVIEW

**Comment by bedroesb** [comment]
Maybe related to: https://github.com/elixir-europe/rdmkit/issues/753

**Comment by lauportell** [comment]
I just reviewed the document and it looks great. My only comment is that it would be good to mention the specific aspects of the use of EHR data for secondary use, which involves using this data for purposes beyond the initial clinical care, such as research or analytics. Like how this data should be adapted for it or if there's need of processing (anonymisation?) It would be good to add information about the HealthData@EU project, for example? This addition would contribute to a more general understanding of the management and ethical use of health data in research studies. Maybe it's out of the scope of this page, let me know what you think.

**Comment by smza** [comment]
> I just reviewed the document and it looks great. My only comment is that it would be good to mention the specific aspects of the use of EHR data for secondary use, which involves using this data for purposes beyond the initial clinical care, like research. For example, how this data should be adapted for it or if there's need of processing (anonymisation?). It would be good to add information about the HealthData@EU project, for example? This would contribute to a general understanding of the management and ethical use of health data in research studies. What do you think?
> 
> If you agree, I can give it a try to add this information.

Hi @lauportell , thanks for your valuable comment. To add this information I think to is best to discuss this with the authors of the page mainly Tim Beck (from Health Data Focus group). I'll write an email and copy you in since I don't have Tim's gitid.

**Comment by lauportell** [comment]
Hi all!
I have added the information about TEHDAS and HealthData@EU in the page. Let me know if you think it fits there. I agree with Tim's comment about leaving the anonymisation piece for the sensitive data page as it is already mentioned there.
Thank you all for the good work!

**Review by lauportell — APPROVED** [review]

**Comment by lauportell** [comment]
I just approved again the PR ( didn't know I could do it as I did some changes :) )
Small things to be done before merging:
1) Add contributors to the contributors yaml
2) Add tools to the tools table and change to the new way of visualizing tools
3) Add page to the sidebar
4) Add news item
@smza will you do it?

**Comment by smza** [comment]
> I just approved again the PR ( didn't know I could do it as I did some changes :) ) Small things to be done before merging:
> 
> 1. Add contributors to the contributors yaml
> 2. Add tools to the tools table and change to the new way of visualizing tools
> 3. Add page to the sidebar
> 4. Add news item
>    @smza will you do it?

Hi @lauportell thanks for your contributions.  I'm waiting from the group for information/description for the tools before adding them to yaml. Once I get that then I'll add the tools to the yaml file.

Cheers
Munazah

File: pages/your_domain/health_data.md
Comment (bianchini88) [inline pages/your_domain/health_data.md:23]:
```suggestion
Patient Reported Outcome Measure (PROM) is a measurement that comes directly from the patient (i.e. study participant) about the status of a patient’s health condition without amendment or interpretation of the patient’s response by a clinician or anyone else.
```

File: pages/your_domain/health_data.md
Comment (bianchini88) [inline pages/your_domain/health_data.md:24]:
```suggestion
Observer-Reported Outcome Measure (ObsROM) is a measurement based on a report of observable signs, events or behaviours related to a patient’s health condition by someone (e.g. family member or caregiver) other than the patient or a health professional.
```

File: pages/your_domain/health_data.md
Comment (bianchini88) [inline pages/your_domain/health_data.md:38]:
```suggestion
* Design the questionnaires with consideration given to interoperability (e.g. mapping questions/answers to concepts in clinical terminologies) and avoiding ambiguity in the questions.
```

File: pages/your_domain/health_data.md
Comment (bianchini88) [inline pages/your_domain/health_data.md:48]:
```suggestion
* Development of the machine-readable data dictionary for all languages:   
```

**Review by bianchini88 — CHANGES_REQUESTED** [review]

**Comment by bedroesb** [comment]
@smza do you have a look a the review from @bianchini88 ?

**Comment by smza** [comment]
> @smza do you have a look a the review from @bianchini88 ?
not yet

@bedroesb I'm waiting for review of some new content that was added by Laura from the team. The team also have to provide the Tools descriptions. probably end of feb

File: pages/your_domain/health_data.md
Comment (bedroesb) [reply pages/your_domain/health_data.md:3]: yes indeed!

**Review by smza — COMMENTED** [review]
Accepted all changes

File: _data/CONTRIBUTORS.yaml
Comment (bedroesb) [inline _data/CONTRIBUTORS.yaml:642]: @smza still to be filled in or you can take them out :)

**Review by bianchini88 — APPROVED** [review]

File: _data/CONTRIBUTORS.yaml
Comment (bianchini88) [inline _data/CONTRIBUTORS.yaml:618]: Most new contributors do not have ORCID.

**Comment by bedroesb** [comment]
Question to all: did we tackle https://github.com/elixir-europe/rdmkit/issues/753 ?

File: pages/your_domain/health_data.md
Comment (bianchini88) [inline pages/your_domain/health_data.md:69]:
```suggestion
* {% tool "eu-survey" %} - SaaS provided by the EU Commission
```

File: pages/your_domain/health_data.md
Comment (bianchini88) [inline pages/your_domain/health_data.md:97]:
```suggestion
EHDS places a significant emphasis on promoting the secondary use of health data, to serve research, innovation, policy-making and regulatory purposes. To achieve this, several European initiatives have been launched to establish its foundation and define guiding principles. Among these endeavors, two signification projects stand out:
```

File: pages/your_domain/health_data.md
Comment (bianchini88) [inline pages/your_domain/health_data.md:108]:
```suggestion
Likely, the use of one of the following standards will not be a choice, but something imposed by the software providers of the EHR systems. These interchange formats are the standards that are likely to be present as part of the EHR, and include:
```

File: _data/CONTRIBUTORS.yaml
Comment (smza) [reply _data/CONTRIBUTORS.yaml:642]: This is all the information I have for the contributors for now

File: pages/your_domain/health_data.md
Comment (smza) [reply pages/your_domain/health_data.md:108]: I would prefer to keep the original text.

Comment (bianchini88) [reply pages/your_domain/health_data.md:108]: I have edited the suggestion above.

File: _data/CONTRIBUTORS.yaml
Comment (smza) [reply _data/CONTRIBUTORS.yaml:618]: Yes thats the way it's going to be for now.  They don't have time to chase the authors and the page is needed live asap!

**Review by smza — COMMENTED** [review]
Review done! All suggestions have been resolved.

**Pull request merged**
